tests/test_parser.py

```diff
@@ -367,7 +368,7 @@ def collect(self):
 
         registry = CollectorRegistry()
         registry.register(TextCollector())
-        self.assertEqual(text.encode('utf-8'), generate_latest(registry))
+        self.assertEqual(text.encode('utf-8'), generate_latest(registry, ALLOWUTF8))
```

Comment:
Also unneeded here (also in various other tests, but these were the files that had the fewest changes).

Reply:
this one is actually needed. the default for generate_latest is underscores, not utf8, and this test contains dots.